feature/experimental/replay/tests/presession_test/presession_test.go

```diff
@@ -29,9 +29,9 @@ func TestMain(m *testing.M) {
 }
 
 func TestReplay(t *testing.T) {
-	const logFile = "https://github.com/openconfig/featureprofiles/raw/main/feature/experimental/replay/tests/presession_test/grpclog.pb"
+	const logFile = "grpclog.pb"
 	t.Logf("Parsing log file: %v", logFile)
-	rec := replayer.ParseURL(t, logFile)
+	rec := replayer.Parse(t, logFile)
```

Comment:
These should be `ParseFile()` https://github.com/openconfig/replayer/blob/main/replayer.go#L45